common/scala/Dockerfile

```diff
@@ -1,15 +1,15 @@
 # Licensed to the Apache Software Foundation (ASF) under one or more contributor
 # license agreements; and to You under the Apache License, Version 2.0.
 
-FROM adoptopenjdk/openjdk8:x86_64-alpine-jdk8u172-b11
+FROM adoptopenjdk/openjdk8-openj9:alpine
```

Comment:
Can you performance test `FROM openjdk:8u201-jdk-alpine3.9` since the JDK has memory issue?

Reply:
@dubee The performance doesnt look promising with this version.

<img width="912" alt="Screen Shot 2019-04-15 at 10 14 35" src="https://user-images.githubusercontent.com/12074169/56117028-863e2280-5f67-11e9-92d5-ba7aee479e60.png">